config-templates/redhat/odf-local/4.8/custom-parameters.json

```diff
@@ -84,6 +99,10 @@
     "name": "cluster-encryption",
     "default": "false",
     "type": "boolean",
+    "regex": "^(?:essentials|advanced)$",
+    "min-length": "8",
+    "max-length": "10",
+    "supported-values": ["essentials", "advanced"],
```

Comment:
These are wrong values. Accespted value looks to be either `true` or `false`

Reply:
Addressed in https://github.com/IBM/ibm-satellite-storage/pull/194/commits/bfad1599711afef7153182a023ded18ea5e91331